src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -311,6 +311,8 @@ public function getParent()
                 throw $e;
             } catch (\Exception $e) {
                 throw new InvalidConfigurationException(sprintf('Invalid configuration for path "%s": %s', $this->getPath(), $e->getMessage()), $e->getCode(), $e);
+            } catch (\Throwable $e) {
```

Comment:
-1 for that. a validation closure is expected to throw an exception on invalid data, not an Error. I suggest leaving Errors as is as they can only happen in a buggy code and so should bubble.